test/smoke.spec.js

```diff
@@ -0,0 +1,53 @@
+var expect = require('chai').expect;
+var sinon = require('sinon');
+var sh = require('shelljs');
+var path = require('path');
+var basePath = path.resolve(__dirname, '..');
+var helper = require('./generateEsLintResult');
+var fs = require('fs-extra');
+var pathToTestJson = path.resolve(__dirname, 'result.json');
+var pathToIndex = path.resolve(__dirname, '..', 'index.js');
+var input = [];
```

Comment:
it isn`t nedd to add otherwise input in before callback because in tests don`t push to array and otherwise it if need to use it